fastlane/docs/Actions.md

```diff
@@ -863,6 +863,14 @@ Additionally you can specify `notes`, `emails`, `groups` and `notifications`.
 
 The following environment variables may be used in place of parameters: `CRASHLYTICS_API_TOKEN`, `CRASHLYTICS_BUILD_SECRET`, and `CRASHLYTICS_FRAMEWORK_PATH`.
 
+### [upload_symbols_to_crashlytics]
+
+This action allows you to upload additional symbolication files to Crashlytics. It's extra useful if you use it to download the latest dSYM files from Apple when you use Bitcode.
```

Comment:
I'm not sure I'd say "additional". That implies to me that some dSYMs are not important. I'd remove "additional" so that people know they should be using this to upload *all* dSYMs.